app/models/group.rb

```diff
@@ -1,3 +1,8 @@
 class Group < ApplicationRecord
+  belongs_to :teacher
   has_many :students_groups
+
+  def to_s
```

Comment:
Cool!! I like it that you made a to string method, really nice idea

Reply:
Yes Carlos told me to use that, actually I wanted to do pluck query myself lol